guides/source/active_record_querying.md

```diff
@@ -59,6 +59,7 @@ To retrieve objects from the database, Active Record provides several finder met
 
 The methods are:
 
+* `annotate`
```

Comment:
Is `annotate` a finder method? Just curious to understand this because the documentation on the above line says 

```
To retrieve objects from the database, Active Record provides several finder methods. Each finder method allows you to pass arguments into it to perform certain queries on your database without writing raw SQL.
``` 

As per the changelog, it says `Add 'ActiveRecord::Relation#annotate' for adding SQL comments to its queries.`

Reply:
Hi, @abhaynikam! It's possible the description should be generalized a bit. Several other relation query methods arguably don't fit that documention sentence literally, either — `preload`, `create_with`, and `includes` being examples.

Perhaps the intro sentence could be updated in another PR.

Reply:
Thanks, @mattyoho .  That makes sense, I agree a few other methods do not fit the description. 

Great addition @mattyoho 🎉

Reply:
About updating the description, I would say let remove methods which do not fit description sinces that particular documentation section is for understanding how to retrieve objects from database. 

I'll like to raise a PR if somebody from Rails team approve it is correct to update the doc 😊